rosdep/python.yaml

```diff
@@ -2912,9 +2912,7 @@ python-psycopg2:
   debian: [python-psycopg2]
   fedora: [python-psycopg2]
   gentoo: [=dev-python/psycopg-2*]
-  ubuntu:
-    '*': [python-psycopg2]
-    focal: [python3-psycopg2]
+  ubuntu: [python-psycopg2]
```

Comment:
CI is failing because it can't find this package in Ubuntu, which I should have guessed.  In that case, I'm going to suggest we just remove the key completely since it isn't available in any of our supported distributions:

```suggestion
```